lsp-mode.el

```diff
@@ -3470,10 +3471,14 @@ in that particular folder."
   (lsp-managed-mode 1)
 
   (run-hooks 'lsp-after-open-hook)
-  (-some-> lsp--cur-workspace
+  (-some->> lsp--cur-workspace
```

Comment:
This is a bit weird structure - can you refactor to `(let ((client ..)` and then the lambda block?

Reply:
Ok